Issue-Label-Bot-v2/notebooks/mlp.py

```diff
@@ -0,0 +1,149 @@
+from sklearn.neural_network import MLPClassifier
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import precision_recall_curve
+import dill as dpickle
+import numpy as np
+import pandas as pd
+import logging
+
```

Comment:
please add requirements.txt file with strictly pinned versions as will make it easier to reproduce your environment.

Reply:
Okay